rezervo/providers/sessions.py

```diff
@@ -16,6 +16,8 @@ def get_user_planned_sessions_from_schedule(
     for d in schedule.days:
         for c in d.classes:
             for cc in chain_config.recurring_bookings:
+                if c.location.id != cc.location_id:
+                    continue
```

Comment:
Nice catch! :hugs: 

I also added the same check to `find_class_in_schedule_by_config` in `rezervo/providers/schedule.py`

Reply:
🤗